### ISSUE TITLE
Fatal error: invalid monster flag KEEP_DISTANCE with Magiclysm on 0.F-1

### ISSUE THREAD

**Issue by wapcaplet** [CleverRaven/Cataclysm-DDA#50828]
### Describe the bug

When starting a new game on 0.F-1 with Magiclysm enabled, a fatal error occurs, preventing the game from loading:

```
 DEBUG    : Error: Json error: data/mods/Magiclysm/monsters/holiday_magiclysm.json:112:66: invalid monster flag: "KEEP_DISTANCE"

ar_triggers": [ "HURT", "FIRE" ],
    "special_attacks": [ { "type": "spell", "spell_data": { "id": "prank", "min_level": 5 }, "cooldown": 80 }, [ "PULL_METAL_WEAPON", 25 ] ],
    "flags": [ "SEES", "HEARS", "SMELLS", "PATH_AVOID_DANGER_1", "
                                                                 ^
                                                                  KEEP_DISTANCE", "WARM", "HIT_AND_RUN" ]
  }
]

 FUNCTION : bool main_menu::new_character_tab()
 FILE     : src/main_menu.cpp
 LINE     : 865
 VERSION  : 0.F-1
```

### Steps To Reproduce

1. Create a new world with Magiclysm enabled
2. Start a new game in that world

### Expected behavior

Expected new game world to load

### Versions and configuration

- OS: Linux
    - OS Version: Distributor ID: Pop; Description: Pop!_OS 20.04 LTS; Release: 20.04; Codename: focal; 
- Game Version: 0.F-1 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Bionic Professions [package_bionic_professions]
]

**Comment by ZhilkinSerg** [comment]
I've force pushed that version with the fix.

Try https://github.com/CleverRaven/Cataclysm-DDA/releases/download/0.F-1/cataclysmdda-0.F-Linux_x64-Curses-0.F-1.tar.gz or building with the 0.F-1 tag again.

**Comment by wapcaplet** [comment]
@ZhilkinSerg I rebuilt from master 0.F-1 to include your commit af757339f ("Add mising flag") but am continuing to see the same `KEEP_DISTANCE` error with `holiday_magiclysm.json`.

**Comment by ZhilkinSerg** [comment]
Magiclysm error should be fixed with 8c7366e861

**Comment by Night-Pryanik** [comment]
No longer happens.
